solution/solution.py

```diff
@@ -7,8 +7,11 @@ def _intro(inputs=None):
 def _basic_arithmetic(inputs=None):
     return [3, 24, 1]
 
+def _basic_if(inputs=None):
```

Comment:
since you're adding the solution for this game, can you update the prompt as well?